ecs-agent/tcs/model/api/api-2.json

```diff
@@ -4,11 +4,12 @@
     "apiVersion":"2015-03-10",
     "endpointPrefix":"ecstcs",
     "jsonVersion":"1.1",
+    "protocol":"json",
     "serviceAbbreviation":"Amazon ECS TCS",
     "serviceFullName":"Amazon EC2 Container Service Telemetry Communication Service",
     "signatureVersion":"v4",
     "targetPrefix":"AmazonEC2ContainerServiceTelemetryV20150310",
-    "protocol":"json"
+    "uid":"ecstcs-2015-03-10"
```

Comment:
Q: is this file generated? Don't remember I see this field in previous generation. Any thing changed from CP side?

Reply:
Yea it's generated automatically.

Reply:
>  Any thing changed from CP side?

The new field is the only one change I'm aware of.